web-api/src/business/useCases/trialSessions/getTrialSessionOpenCasesCountInteractor.ts

```diff
@@ -0,0 +1,42 @@
+import { NotFoundError, UnauthorizedError } from '@web-api/errors/errors';
+import {
+  ROLE_PERMISSIONS,
+  isAuthorized,
+} from '@shared/authorization/authorizationClientService';
+import { applicationContext } from '@web-api/applicationContext';
+import { UnknownAuthUser } from '@shared/business/entities/authUser/AuthUser';
+import { getCasesInTrialSession } from '@web-api/business/useCases/trialSessions/updateTrialSessionInteractorHelper';
+import { getTrialSessionById } from '@web-api/persistence/dynamo/trialSessions/getTrialSessionById';
+
+export const getTrialSessionOpenCasesCountInteractor = async (
```

Comment:
It feels like the name of this interactor isn't quite in line with what it does.